File: app/src/test/java/com/pij/zworkout/workout/StateTestUtil.java
package com.pij.zworkout.workout;

import com.pij.zworkout.uc.Workout;

/**
 * <p>Created on 12/03/2018.</p>
 *
 * @author Pierrejean
 */
public class StateTestUtil {

    public static State empty() {
        return new State(false, null, false, Workout.Companion.getEMPTY(), true, null);
    }
}

File: app/src/test/java/com/pij/zworkout/workout/StateTestUtil.kt
package com.pij.zworkout.workout

import com.pij.zworkout.uc.Workout

/**
 *
 * Created on 12/03/2018.
 *
 * @author Pierrejean
 */
object StateTestUtil {

    fun empty(): State {
        return State(false, null, false, Workout.EMPTY, true, null)
    }
}